learning_journal/tests/test_learning_journal.py

```diff
@@ -0,0 +1,47 @@
+from learning_journal.models import Entry
+
+
+def test_entry_creation(session):
+    test_entry = Entry(title=u"Test Entry", text=u"Here is my test entry")
+    session.add(test_entry)
+    session.flush()
+    assert session.query(Entry).filter(Entry.title==u"Test Entry")
+
+
+def test_list_view(app, one_entry):
+    response = app.get('/')
+    assert response.status_code == 200
+    actual = response.body
+    assert one_entry.title.encode('utf8') in actual
```

Comment:
you might try accessing the `response.text` attribute, which I believe is the rendered HTML as unicode.